.gitlab/ci/.gitlab-ci.sdk-nightly.yml

```diff
@@ -197,60 +286,36 @@ demisto-sdk-nightly:run-commands-against-instance:
     ARTIFACTS_FOLDER_INSTANCE: "${ARTIFACTS_FOLDER}/instance_${INSTANCE_ROLE}"
     ARTIFACTS_FOLDER_SERVER_TYPE: "${ARTIFACTS_FOLDER}/server_type_${SERVER_TYPE}"
     ENV_RESULTS_PATH: "${ARTIFACTS_FOLDER_SERVER_TYPE}/env_results.json"
+    E2E_TEST_FOLDER: "general"
   needs: ["demisto-sdk-nightly:xsoar-prepare-testing-bucket"]
   stage: run-instances
   script:
     - !reference [.ssh-config-setup]
-    - section_start "Wait Until Server Ready"
-    - |
-      [ -n "${NIGHTLY}" ] && IS_NIGHTLY=true || IS_NIGHTLY=false
-      python3 ./Tests/scripts/wait_until_server_ready.py  -n ${IS_NIGHTLY} --instance-role "${INSTANCE_ROLE}" || EXIT_CODE=$?
-    - section_end "Wait Until Server Ready"
-    - !reference [.download-demisto-conf]
-    - !reference [.secrets-fetch]
+
+    - section_start "End to End Tests"  --collapsed
     - |
+      mkdir $ARTIFACTS_FOLDER/demisto-sdk
+      git clone -b ${SDK_REF} --single-branch --depth 1 https://github.com/demisto/demisto-sdk.git $ARTIFACTS_FOLDER/demisto-sdk
```

Comment:
why do you clone the git again here, it's best to clone it just once in a share common step and use it from the artifacts folder.

Reply:
We will not clone it more than 1 time each